ci/docker/connector_build/Dockerfile

```
@@ -1,9 +1,11 @@
 # We use manylinux1 base image because pyarrow_manylinux2010 has a bug and wheel failed to be audited
-ARG BASE_IMAGE=quay.io/pypa/manylinux2010_x86_64
+ARG BASE_IMAGE
```

Please keep the default values
```suggestion
ARG BASE_IMAGE=quay.io/pypa/manylinux2010_x86_64
```